src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -1513,6 +1513,25 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    validator=get_permission_validator(t_share_permissions))
         c.ignore('sas_token')
 
+    with self.argument_context('storage share list-handle') as c:
+        c.register_path_argument(default_file_param="")
+        c.argument('recursive', arg_type=get_three_state_flag())
```

Comment:
Can you add `show_next_marker` argumenet so that customer can get the `marker` from cli output and keep paging. See examples:
https://github.com/Azure/azure-cli/blob/f6f34721e9a59882dbe8a9031f05a1b4483c6d1d/src/azure-cli/azure/cli/command_modules/storage/_params.py#L1364-L1365
https://github.com/Azure/azure-cli/blob/f6f34721e9a59882dbe8a9031f05a1b4483c6d1d/src/azure-cli/azure/cli/command_modules/storage/operations/blob.py#L194-L200

Reply:
The result returned by track1 sdk provides next_marker by default.